tests/Parsers/ImagePickerParserTest.php

```diff
@@ -0,0 +1,92 @@
+ * @uses \Collecthor\SurveyjsParser\Variables\SingleChoiceVariable
+ * @uses \Collecthor\SurveyjsParser\Values\StringValueOption
+ * @uses \Collecthor\SurveyjsParser\Values\IntegerValueOption
+ * @uses \Collecthor\SurveyjsParser\Traits\GetDisplayValue
+ * @uses \Collecthor\SurveyjsParser\SurveyConfiguration
+ * @uses \Collecthor\SurveyjsParser\Variables\OpenTextVariable
+ */
+final class ImagePickerParserTest extends TestCase
+{
+    public function testParseSingleImagePicker(): void
+    {
+        $surveyConfig = new SurveyConfiguration(locales:['default', 'nl']);
+        $questionConfig = [
+            "type" => "imagepicker",
+            "name" => "question3",
+            "choices" => [
+                [
+                    "value" => "lion",
+                    "imageLink" => "https://surveyjs.io/Content/Images/examples/image-picker/lion.jpg"
+                ],
+                [
+                    "value" => "giraffe",
+                    "imageLink" => "https://surveyjs.io/Content/Images/examples/image-picker/giraffe.jpg"
+                ],
+                [
+                    "value" => "panda",
+                    "imageLink" => "https://surveyjs.io/Content/Images/examples/image-picker/panda.jpg"
+                ],
+                [
+                    "value" => "camel",
+                    "imageLink" => "https://surveyjs.io/Content/Images/examples/image-picker/camel.jpg"
+                ]
+            ]
+        ];
+
+        $parser = new ImagePickerParser();
+        $result = toArray($parser->parse(new DummyParser(), $questionConfig, $surveyConfig));
+
+        self::assertInstanceOf(SingleChoiceVariable::class, $result[0]);
+    }
+
+    public function testParseMultipleImagePicker(): void
+    {
+        $surveyConfig = new SurveyConfiguration(locales:['default', 'nl']);
+        $questionConfig = [
+            "type" => "imagepicker",
+            "name" => "question3",
+            "choices" => [
+                [
+                    "value" => "lion",
+                    "imageLink" => "https://surveyjs.io/Content/Images/examples/image-picker/lion.jpg"
+                ],
+                [
+                    "value" => "giraffe",
+                    "imageLink" => "https://surveyjs.io/Content/Images/examples/image-picker/giraffe.jpg"
+                ],
+                [
+                    "value" => "panda",
+                    "imageLink" => "https://surveyjs.io/Content/Images/examples/image-picker/panda.jpg"
+                ],
+                [
+                    "value" => "camel",
+                    "imageLink" => "https://surveyjs.io/Content/Images/examples/image-picker/camel.jpg"
+                ],
+            ],
+            "multiSelect" => true,
+        ];
+
+        $parser = new ImagePickerParser();
+        $result = toArray($parser->parse(new DummyParser(), $questionConfig, $surveyConfig));
+
+        self::assertInstanceOf(MultipleChoiceVariable::class, $result[0]);
```

Comment:
This test is not proper; test the actual options as well as their display values.
(I'm fairly confident that display values currently don't work very well here since they don't take imagelink into account)